test/performance/cluster_information.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Only `run.sh` is using this script, and there's no mention of its use in the readme. Would it be worth including it in `run.sh`? After all, it's actually just 2 lines of (useful) code.

Reply:
I intended it to be a "general purpose" script to be run before each performance test to gather all useful information about the cluster the tests are run on. I envisioned this to eventually grow and be useful across perf. tests. I can move it for now though for sure.